lib/style/pipes.ex

```diff
@@ -143,17 +143,23 @@ defmodule Styler.Style.Pipes do
     quote do: {:|>, _, [{:|>, _, [unquote(a), unquote(b)]}, unquote(c)]}
   end
 
+  # Unary operators needs to be expanded (#128)
+  defp maybe_wrap_unary({:-, meta, []}), do: {{:., meta, [{:__aliases__, meta, [:Kernel]}, :-]}, meta, []}
```

Comment:
```suggestion
  defp maybe_wrap_unary({op, meta, []}) when op in [:-, :+], do: {{:., meta, [{:__aliases__, meta, [:Kernel]}, op]}, meta, []}
```

The only other unary is `+/1`, as per https://hexdocs.pm/elixir/operators.html#general-operators

Reply:
I didn't handle it because I didn't know what the + operator was used for :sweat_smile: But yeah for the sake of completeness I can def add it